python/lsst/afw/cameraGeom/utils.py

```diff
@@ -156,18 +177,31 @@ def plotFocalPlane(camera, fieldSizeDeg_x=0, fieldSizeDeg_y=None, dx=0.1, dy=0.1
 
 def makeImageFromAmp(amp, imValue=None, imageFactory=afwImage.ImageU, markSize=10, markValue=0,
                      scaleGain = lambda gain: (gain*1000)//10):
-    """!Make an image from an amp object
-
-    Since images are integer images by default, the gain needs to be scaled to give enough dynamic range
-    to see variation from amp to amp.  The scaling algorithm is assignable.
-
-    @param[in] amp  Amp record to use for constructing the raw amp image
-    @param[in] imValue  Value to assign to the constructed image scaleGain(gain) is used if not set
-    @param[in] imageFactory  Type of image to construct
-    @param[in] markSize  Size of mark at read corner in pixels
-    @param[in] markValue  Value of pixels in the read corner mark
-    @param[in] scaleGain  The function by which to scale the gain
-    @return an untrimmed amp image
+    """Make an image from an amp object.
+
+    Since images are integer images by default, the gain needs to be scaled to
+    give enough dynamic range to see variation from amp to amp.
+    The scaling algorithm is assignable.
+
+    Parameters
+    ----------
+    amp : `lsst.afw.table.AmpInfoRecord`
+        Amp record to use for constructing the raw amp image
+    imValue : `float` or None
+        Value to assign to the constructed image, or scaleGain(gain) if None.
+    imageFactory : `lsst.afw.image.Image`
```

Comment:
Is this the right type? It's supposed to be a class, so shouldn't it be `type`? Or `subclass of lsst.afw.image.Image`?

Reply:
Interesting point. I guess it's technically a `callable`, no?

How about ```imageFactory : callable like `lsst.afw.image.Image````?